mappings/net/minecraft/client/gui/widget/LevelListWidget.mapping

```diff
@@ -11,6 +11,11 @@ CLASS dcr net/minecraft/client/gui/widget/LevelListWidget
 			ARG 2 levelList
 			ARG 3 level
 			ARG 4 levelStorage
+		METHOD a play ()V
```

Comment:
Hmm - what do these methods do?

Reply:
I think they're called when the user clicks play, for example.

Reply:
Correct. It's what gets executed, when you press `play` on the level select screen.